pkg/backends/calico/client.go

```diff
@@ -165,12 +166,19 @@ func NewCalicoClient(confdConfig *config.Config) (*client, error) {
 			typhaConnection.Finished.Wait()
 			log.Fatal("Connection to Typha failed")
 		}()
+
+		// Allow a little time before returning, for Typha to connect and data to
+		// start flowing, so that an initial snapshot of the datastore will be in
+		// place when we return.
+		log.Info("Allow time for initial datastore snapshot")
+		time.Sleep(250 * time.Millisecond)
```

Comment:
Could you add a brief comment to explain why you chose 250ms? What would "break" if it was really short (e.g. not enough time would pass for data to arrive from typha) and what would "break" if it was really long (e.g. GR would break because confd wouldn't allow bird to restart in time)

And is there no way to "fix" this without a sleep? e.g. block until data has started arriving from Typha.